src/client/caseCorrect.js

```diff
@@ -1,7 +1,8 @@
 var yggdrasil = require('yggdrasil')({});
 var UUID = require('uuid-1345');
 
-module.exports = function(client, options) {
+module.exports = function(client) {
+  var options = client.options;
```

Comment:
Ah I saw your commit message "This allows client plugins to change the options on the client object,
and have it reflected in other plugins. Previously, had to rely on the
options being the same object, and hold a reference to it. Now it is
always accessible as `client.options`."

It sounds like it assume some order of execution between plugins. Which plugins is accessing the modified property ?